launcher/ui/widgets/ModFilterWidget.cpp

```diff
@@ -321,7 +320,8 @@ void ModFilterWidget::setCategories(const QList<ModPlatform::Category>& categori
 {
     m_categories = categories;
 
-    delete ui->categoryGroup->layout();
+    if (ui->categoryGroup->layout())
+        delete ui->categoryGroup->layout();
```

Comment:
Redundant change? Deleting a nullptr should just do... nothing

Reply:
Removed it. still strange that that delete line was mentioned in the crash log provided in the issue.
I hope the crash was only because of the double deletion of the dialog.